src/storage/idb/index.js

```diff
@@ -18,6 +19,9 @@ const init = () => {
         case 0:
           db.createObjectStore(IPScanJobsMeta, { autoIncrement: true });
           db.createObjectStore(IPScanJobsData, { autoIncrement: true });
+        // eslint-disable-next-line no-fallthrough
+        case 1:
```

Comment:
So this way, when someone that already has a version of InterPro running, gets only the update of the version 2.